src/main/java/pl/isa/biblioteka/books/BorrowBooks.java

```diff
@@ -0,0 +1,84 @@
+package pl.isa.biblioteka.books;
+
+import pl.isa.biblioteka.user.LogUser;
+import pl.isa.biblioteka.user.Person;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class BorrowBooks {
+    public static Scanner scanner = new Scanner(System.in);
+    public static BooksEdit booksEdit = new BooksEdit();
+
+    public static LogUser logUser = new LogUser();
+
+    public static boolean addBookToPerson() {
+        List<Book> booksList = BooksEdit.booksList;
+        System.out.println("Podaj tytuł książki do wypożyczenia: ");
```

Comment:
Dalej nie do końca to o czym myślałem, bookTitle powinno przychodzić jako parametr z menu, a nie być wczytywane tutaj.

Reply:
przeniesione do menu